pyseir/cli.py

```diff
@@ -156,7 +150,10 @@ def _build_all_for_states(
 
     with Pool(maxtasksperchild=1) as p:
         # calculate calculate county inference
-        p.map(infer_rt.run_rt_for_fips, all_county_fips.keys())
+        infer_rt_inputs = [
+            infer_rt.RegionalInput.from_fips(fips) for fips in all_county_fips.keys()
```

Comment:
fwiw you can just do `for fips in all_county_fips` here

Reply:
True, but I find the default of iterating on keys instead of items not obvious so prefer to be explicit.